src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -17,6 +17,8 @@ public static class ScriptConstants
         public const string AzureFunctionsColdStartKey = "MS_AzureFunctionsColdStart";
         public const string AzureFunctionsHttpRequestKey = "MS_AzureFunctionsHttpRequest";
         public const string AzureProxyFunctionExecutorKey = "MS_AzureProxyFunctionExecutor";
+        public const string AzureProxyFunctionLocalRedirectHeader = "MS_AzureProxyFunctionLocalRedirectHeader";
+        public const int AzureProxyFunctionMaxLocalRedirects = 10;
```

Comment:
Are there valid scenarios for people having nested redirects this deep? What if the max were 5 instead?

Reply:
Yes this is valid scenario. And as I put in the comments I am just mimicking the IIS ARR behavior which default is 10.